quirks/password-rules.json

```
@@ -326,6 +332,9 @@
     "marriott.com": {
         "password-rules": "minlength: 8; maxlength: 20; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$!#&@?%=];"
     },
+    "maybank2u.com.my": {
+        "password-rules": "minlength: 8; maxlength: 12; max-consecutive: 2; required: lower, upper, digit, special;"
```

Similar to the above, the `required` property needs to be split up here (to account for the **and** relationship in the site's password rule information).

```
minlength: 8; maxlength: 12; max-consecutive: 2; required: lower; required: upper; required: digit; required: special;
```

---

Implemented your suggestion:
```suggestion
        "password-rules": "minlength: 8; maxlength: 12; max-consecutive: 2; required: lower, upper, digit, special;"
```